### PR TITLE
GT Updates with GEM electronics map, hadronic PF calibrations, JECs for 2017 MC

### PR DESCRIPTION
**Pull request by arunhep**
# Summary of changes in Global Tags 
 
## RunI simulation 
 
   * **RunI Heavy Ions scenario** : [101X_mcRun1_HeavyIon_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_HeavyIon_v2) as [101X_mcRun1_HeavyIon_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_HeavyIon_v1) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun1_HeavyIon_v2/101X_mcRun1_HeavyIon_v1): 
      * update of hadronic PF calibrations

   * **RunI Ideal scenario** : [101X_mcRun1_design_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_design_v2) as [101X_mcRun1_design_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_design_v1) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun1_design_v2/101X_mcRun1_design_v1): 
      * update of hadronic PF calibrations

   * **RunI Proton-Lead scenario** : [101X_mcRun1_pA_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_pA_v2) as [101X_mcRun1_pA_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_pA_v1) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun1_pA_v2/101X_mcRun1_pA_v1): 
      * update of hadronic PF calibrations

   * **RunI Realistic scenario** : [101X_mcRun1_realistic_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_realistic_v2) as [101X_mcRun1_realistic_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun1_realistic_v1) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun1_realistic_v2/101X_mcRun1_realistic_v1): 
      * update of hadronic PF calibrations

## RunII simulation 
 
   * **RunII Asymptotic scenario** : [101X_mcRun2_asymptotic_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_asymptotic_v3) as [101X_mcRun2_asymptotic_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_asymptotic_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun2_asymptotic_v3/101X_mcRun2_asymptotic_v2): 
      * update of hadronic PF calibrations

   * **RunII CRAFT scenario** : [101X_mcRun2cosmics_startup_deco_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2cosmics_startup_deco_v3) as [101X_mcRun2cosmics_startup_deco_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2cosmics_startup_deco_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun2cosmics_startup_deco_v3/101X_mcRun2cosmics_startup_deco_v2): 
      * update of hadronic PF calibrations

   * **RunII Heavy Ions scenario** : [101X_mcRun2_HeavyIon_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_HeavyIon_v4) as [101X_mcRun2_HeavyIon_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_HeavyIon_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun2_HeavyIon_v4/101X_mcRun2_HeavyIon_v3): 
      * update of hadronic PF calibrations

   * **RunII Ideal scenario** : [101X_mcRun2_design_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_design_v3) as [101X_mcRun2_design_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_design_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun2_design_v3/101X_mcRun2_design_v2): 
      * update of hadronic PF calibrations

   * **RunII Proton-Lead scenario** : [101X_mcRun2_pA_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_pA_v3) as [101X_mcRun2_pA_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_pA_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun2_pA_v3/101X_mcRun2_pA_v2): 
      * update of hadronic PF calibrations

   * **RunII Startup scenario** : [101X_mcRun2_startup_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_startup_v3) as [101X_mcRun2_startup_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mcRun2_startup_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mcRun2_startup_v3/101X_mcRun2_startup_v2): 
      * update of hadronic PF calibrations

## RunI data 
 
   * **RunI HLT processing** : [101X_dataRun2_HLT_frozen_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLT_frozen_v5) as [101X_dataRun2_HLT_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLT_frozen_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_HLT_frozen_v5/101X_dataRun2_HLT_frozen_v4): 
      * update of hadronic PF calibrations
      * update of SiPixel 0T templates

   * **RunI Offline processing** : [101X_dataRun2_v7](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_v7) as [101X_dataRun2_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_v7/101X_dataRun2_v6): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of LumiPCC

## RunII data 
 
   * **RunII HLTHI processing** : [101X_dataRun2_HLTHI_frozen_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLTHI_frozen_v6) as [101X_dataRun2_HLTHI_frozen_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLTHI_frozen_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_HLTHI_frozen_v6/101X_dataRun2_HLTHI_frozen_v5): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of SiPixel 0T tag

   * **RunII HLT relval processing** : [101X_dataRun2_HLT_relval_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLT_relval_v6) as [101X_dataRun2_HLT_relval_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLT_relval_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_HLT_relval_v6/101X_dataRun2_HLT_relval_v5): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of SiPixel 0T tag

   * **RunII Offline processing** : [101X_dataRun2_v7](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_v7) as [101X_dataRun2_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_v7/101X_dataRun2_v6): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of LumiPCC

   * **RunII Offline relval processing** : [101X_dataRun2_relval_v7](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_relval_v7) as [101X_dataRun2_relval_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_relval_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_relval_v7/101X_dataRun2_relval_v6): 
      * update of hadronic PF calibrations
      * update of GEM electronics map

   * **RunII Prompt processing** : [101X_dataRun2_PromptLike_v7](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_PromptLike_v7) as [101X_dataRun2_PromptLike_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_PromptLike_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_PromptLike_v7/101X_dataRun2_PromptLike_v6): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of SiPixel 0T tag

   * **RunI HLT processing** : [101X_dataRun2_HLT_frozen_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLT_frozen_v5) as [101X_dataRun2_HLT_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_dataRun2_HLT_frozen_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_dataRun2_HLT_frozen_v5/101X_dataRun2_HLT_frozen_v4): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of SiPixel 0T tag

## Upgrade 
 
   * **PhaseII realistic scenario** : [101X_upgrade2023_realistic_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2023_realistic_v3) as [101X_upgrade2023_realistic_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2023_realistic_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_upgrade2023_realistic_v3/101X_upgrade2023_realistic_v2): 
      * update of hadronic PF calibrations
      * update of GEM electronics map

   * **PhaseI 2018 cosmics scenario** : [101X_upgrade2018cosmics_realistic_deco_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2018cosmics_realistic_deco_v5) as [101X_upgrade2018cosmics_realistic_deco_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2018cosmics_realistic_deco_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_upgrade2018cosmics_realistic_deco_v5/101X_upgrade2018cosmics_realistic_deco_v4): 
      * update of GEM electronics map
      * update of hadronic PF calibrations

   * **PhaseI 2018 design scenario** : [101X_upgrade2018_design_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2018_design_v5) as [101X_upgrade2018_design_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2018_design_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_upgrade2018_design_v5/101X_upgrade2018_design_v4): 
      * update of hadronic PF calibrations
      * update of GEM electronics map

   * **PhaseI 2018 realistic scenario** : [101X_upgrade2018_realistic_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2018_realistic_v4) as [101X_upgrade2018_realistic_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_upgrade2018_realistic_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_upgrade2018_realistic_v4/101X_upgrade2018_realistic_v3): 
      * update of GEM electronics map
      * update of hadronic PF calibrations

   * **PostLS2 design scenario** : [101X_postLS2_design_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_postLS2_design_v3) as [101X_postLS2_design_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_postLS2_design_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_postLS2_design_v3/101X_postLS2_design_v2): 
      * update of hadronic PF calibrations
      * update of GEM electronics map

   * **PostLS2 realistic scenario** : [101X_postLS2_realistic_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_postLS2_realistic_v3) as [101X_postLS2_realistic_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_postLS2_realistic_v2) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_postLS2_realistic_v3/101X_postLS2_realistic_v2): 
      * update of GEM electronics map
      * update of hadronic PF calibrations

## 2017_MC 
 
   * **PhaseI 2017 cosmics peak scenario** : [101X_mc2017cosmics_realistic_peak_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017cosmics_realistic_peak_v5) as [101X_mc2017cosmics_realistic_peak_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017cosmics_realistic_peak_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mc2017cosmics_realistic_peak_v5/101X_mc2017cosmics_realistic_peak_v4): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of JECs

   * **PhaseI 2017 cosmics scenario** : [101X_mc2017cosmics_realistic_deco_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017cosmics_realistic_deco_v5) as [101X_mc2017cosmics_realistic_deco_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017cosmics_realistic_deco_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mc2017cosmics_realistic_deco_v5/101X_mc2017cosmics_realistic_deco_v4): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of JECs

   * **PhaseI 2017 design scenario** : [101X_mc2017_design_IdealBS_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017_design_IdealBS_v5) as [101X_mc2017_design_IdealBS_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017_design_IdealBS_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mc2017_design_IdealBS_v5/101X_mc2017_design_IdealBS_v4): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of JECs

   * **PhaseI 2017 realistic scenario** : [101X_mc2017_realistic_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017_realistic_v5) as [101X_mc2017_realistic_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/101X_mc2017_realistic_v4) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/101X_mc2017_realistic_v5/101X_mc2017_realistic_v4): 
      * update of hadronic PF calibrations
      * update of GEM electronics map
      * update of JECs

### CODE REVIEW

**Comment by arunhep** [comment]
@cmsbuild please test

**Comment by arunhep** [comment]
the build errors are unrelated to this PR.
Let me try again with the tests.

**Comment by arunhep** [comment]
@cmsbuild please test

**Comment by fabiocos** [comment]
please test

**Comment by perrotta** [comment]
All these errors are due to
Global Tag "93X_mcRun2_asymptotic_v5" has not been found in the database

**Comment by arunhep** [comment]
@perrotta 
thanks for pointing it out. i changed the version of frozen GT by mistake. corrected now.

**Comment by arunhep** [comment]
@cmsbuild please test

**Comment by arunhep** [comment]
+1

**Comment by fabiocos** [comment]
@arunhep I notice in the comparison differences in Run1 workflows, these are justified by the updates there I assume. This PR also includes *all* the updates inserted in 9_4_MAOD_X, right?

**Comment by arunhep** [comment]
@fabiocos 

> I notice in the comparison differences in Run1 workflows, these are justified by the updates there I assume. This PR also includes all the updates inserted in 9_4_MAOD_X, right?

yes because PF hadronic calibration updates also entered in Run1 workflows.
and yes the JECs which entered 9_4_MAOD_X are also part of this PR.

**Comment by fabiocos** [comment]
+1
